### ISSUE TITLE
Constants don't convert units properly

### ISSUE THREAD

**Issue by neffmallon** [wright-group/PyCMDS#58]
When using constants, the display units must be correct and all the same for the OPAs or the constant constraints don't work.

**Comment by untzag** [comment]
Can you say more?

**Comment by p770193** [comment]
Presumably the somatic system is reading the value without passing in a `units` kwarg

**Comment by darienmorrow** [comment]
I will add that I have noticed "phantom constants". By this, I mean: say I did a scan in which I used a constant for `wm`. Then I do a scan which includes scanning `wm` (this may entail removing the constant). Then I do a scan of say `[w1, w2]` without specifying the constant for `wm`. Sometimes `wm` reverts to its initial constant and is moved during the `[w1, w2]` scan. If I then try to add a constant for `wm` on the NEXT scan I get an error on the command line which doesn't allow me to enqueue an acquisition. If I remove that desired constant and then click `remove` AGAIN (even though there is no constant displayed) then everything works as expected.
